models/git/branches.go

```diff
@@ -487,7 +487,7 @@ func RemoveDeletedBranchByID(repoID, id int64) (err error) {
 	if affected, err := db.GetEngine(db.DefaultContext).Delete(deletedBranch); err != nil {
 		return err
 	} else if affected != 1 {
-		return fmt.Errorf("remove deleted branch ID(%v) failed", id)
+		return fmt.Errorf("remove deleted branch ID(%w) failed", id)
```

Comment:
But it's wrong here ........ and many other places.

Reply:
Oh yeah, you are absolutely right.
Wait a second, I'll try a more restrictive search.